samples/cellular/nrf_cloud_multi_service/sample.yaml

```diff
@@ -84,7 +84,7 @@ tests:
       - nrf9151dk/nrf9151/ns
       - nrf9160dk/nrf9160/ns
       - nrf9161dk/nrf9161/ns
-    extra_args: nrf_cloud_multi_service_SHIELD=nrf7002ek
```

Comment:
Why the switch to `nrf7002ek_nrf7000`?

AFAIK this will place the `7002` in `nrf7000` emulation mode, won't it? Is that intentional? Am I mistaken?

Reply:
I see now the commit message: `With latest upstream, scan only mode now only works with nRF7000.`

But I find this hard to believe, it really does not work with the 7002?

Reply:
@krish2718 I should have pinged, can you elaborate on this? I guess I am just surprised to see this requirement

Reply:
This change was done as part of the code review process that we got in upstream, the default mode selection is modified,  and because of that the hard preference to nRF7000 for scan only was made, but as it can be emulated using nRF7002EK, it's just really a board name change.

Reply:
Alright, I still don't fully understand, but it sounds like it definitely is intentional.

I think docs changes are required for it, though. Specifically the build commands for scan-only builds

Reply:
Yes, I tried to do the doc changes as well, but please review and let me know in case I have missed something.

> Alright, I still don't fully understand, but it sounds like it definitely is intentional. Can you link the review where this was discussed?

I tried to link it earlier but couldn't find the exact convo but the PR is https://github.com/zephyrproject-rtos/zephyr/pull/75714 specifically this line https://github.com/zephyrproject-rtos/zephyr/pull/75714/files#diff-3444d04d691d3352650ae5af74eefe1cf31b5f611a6bde4805fddeb2847fc077R52 but I am afraid the convo is lost.

Reply:
Hmm, I cannot follow that second link, and I also was unable to find any discussion of this specific issue in the first link. But I will let it rest, since it is noncritical that I understand.


> Yes, I tried to do the doc changes as well, but please review and let me know in case I have missed something.

Ah, yes, in https://github.com/nrfconnect/sdk-nrf/pull/16753 ? I see that I reviewed that earlier, but somehow I missed this specific change. I am not sure how that happened, I apologize. I will update my review.